index.html

```diff
@@ -3,87 +3,65 @@
 isFrontPage: true
 ---
 
-<div class="feature mt-content">
+<div class="feature">
+
+  {% assign image = "public/img/collage.png" | relative_url %}
   {% include components/hero.html
     header = "Welcome to the National Center for Charitable Statistics (NCCS) open data platform"
     description = "Consectetur atque culpa ad quidem ex? Deleniti atque error necessitatibus cumque quisquam. Fugit necessitatibus commodi eveniet."
     ctaUrl = "#"
     ctaText = "Hero CTA Link"
     ctaIcon = "external"
-    imageUrl = "https://placehold.co/855x555"
+    imageUrl = image
     imageAlt = "A11y description"
   %}
 </div>
 
-<div class="content text-center">
-  <h2>Getting Started with NCCS Data</h2>
-  <p>Consectetur neque culpa officiis saepe adipisci. In corporis earum assumenda consequatur voluptatibus.</p>
+<div class="content text-center flow-4">
+  {% include components/content-feature.html
+    header = "Nullam quis risus eget urna mollis"
+    description = "Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum."
+    ctaUrl = "#"
+    ctaText = "Learn how to use NCCS data"
+    imageUrl = "https://placehold.co/855x555"
+    imageAlt = "A11y description"
+  %}
 </div>
 
-<h2>Datasets</h2>
+<div class="filter full flow-4" data-module="filter">
 
-<p>Elit magni temporibus excepturi nemo voluptatibus Illum consequatur accusantium dolorum magni sapiente dolores. Fuga blanditiis ea eius enim itaque. Impedit deserunt excepturi possimus sapiente provident. Quis magni repellendus minus error.</p>
-
-<div class="posts">
   {% for post in site.datasets %}
-  <div class="post">
-    <h2 class="post-title">
-      <a href="{{ post.url | relative_url }}">
-        {{ post.title }}
-      </a>
-    </h2>
-    <span class="post-date">{{ post.date | date_to_string }}</span>
-  </div>
-  {% endfor %}
-</div>
-
-<h2>Projects</h2>
-
-<div class="projects">
-  {% for post in site.projects %}
-  <div class="post">
-    <h2 class="post-title">
-      <a href="{{ post.url | relative_url }}">
-        {{ post.title }}
-      </a>
-    </h2>
-    <span class="post-date">{{ post.date | date_to_string }}</span>
-  </div>
+    {% for category in post.categories %}
+      {% capture all_categories %}
+        {% if all_categories %}
+          {{ all_categories | join: "," }},{{ category }}
+        {% else %}
+          {{ category }}
+        {% endif %}
+      {% endcapture %}
+    {% endfor %}
```

Comment:
This loop sucks all the categories out of the `datasets` collection. However some weird stuff happens with whitespace in these arrays. My hunch is that the `join` plus the `uniq` filter I use later to scrub duplicates leaves spaces rather than truly deleting...